auction9-frontend/src/app/data-table/data-table.component.ts

```diff
@@ -29,32 +30,24 @@ export interface AuctionData {
 })
 
 export class DataTableComponent implements AfterViewInit {
+  // data from parent
+  @Input() auctionsArray : AuctionData[];
```

Comment:
Let's call this in some generic way, for example `data`, it should not be related to auctions at all. All should be generic.

Reply:
Renamed to generic name 'data'